packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.test.tsx

```diff
@@ -207,6 +207,8 @@ describe('VerticalBarChart - mouse events', () => {
     wrapper.update();
 
     wrapper.find('rect').at(1).simulate('mouseover');
+    await new Promise(resolve => setTimeout(resolve));
```

Comment:
happening for all charts not only vertical bar chart. Looks like it is the first render cycle snapshot

Reply:
Fixed.